customTests/stream/Makefile

```diff
@@ -8,7 +8,6 @@
 CFLAGS= -Ofast -mcmodel=large -ffp-contract=fast -lomp -fopenmp -fnt-store=aggressive -DNTIMES=20
 OTHER_FLAGS= -mavx512vl -mavx512bf16 -mavx512vbmi -mavx512vbmi2 -mavx512vnni -mavx512bitalg -mavx512vpopcntdq 
 ZEN4FLAGS= -mavx512f -DSTREAM_ARRAY_SIZE=2400000000 -march=znver4
```

Comment:
So for this particular array size mavx512f is better than just mavx2?

Reply:
I believe you had tested out that array size first and I used it because I was not getting the same perf you were.  I am not sure it makes that big a difference. The expected values were based on this.

Reply:
addressed